### PR TITLE
Upgrade terraform-provider-tailscale to v0.13.13

### LINKED ISSUE #280: Upgrade terraform-provider-tailscale to v0.13.13
Release details: https://github.com/tailscale/terraform-provider-tailscale/releases/tag/v0.13.13

### PR DESCRIPTION
**Pull request by iwahbe**
This PR was generated via `$ upgrade-provider pulumi/pulumi-tailscale`.

---

- Upgrading terraform-provider-tailscale from 0.13.12  to 0.13.13.
	Fixes #280

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -78,39 +47,26 @@ func Provider() tfbridge.ProviderInfo {
...
-			"tailscale_device_authorization": {Tok: makeResource(mainMod, "DeviceAuthorization")},
-			"tailscale_device_tags":          {Tok: makeResource(mainMod, "DeviceTags")},
-			"tailscale_tailnet_key":          {Tok: makeResource(mainMod, "TailnetKey")},
-			"tailscale_device_key":           {Tok: makeResource(mainMod, "DeviceKey")},
 		},
 		DataSources: map[string]*tfbridge.DataSourceInfo{
-			"tailscale_device":  {Tok: makeDataSource(mainMod, "getDevice")},
-			"tailscale_devices": {Tok: makeDataSource(mainMod, "getDevices")},
-			"tailscale_4via6":   {Tok: makeDataSource(mainMod, "get4Via6")},
+			"tailscale_4via6": {Tok: tfbridge.MakeDataSource(mainPkg, mainMod, "get4Via6")},
```
Comment (guineveresaenger) [inline provider/resources.go:69]: do we want to add a comment here that the token computation isn't handling this as expected due to the 4, and therefore it must remain explicit?

File: provider/resources.go [hunk 2]
```diff
@@ -37,39 +37,8 @@ const (
...
-func makeResource(mod string, res string) tokens.Type {
-	fn := string(unicode.ToLower(rune(res[0]))) + res[1:]
-	return makeType(mod+"/"+fn, res)
-}
-
-// preConfigureCallback is called before the providerConfigure function of the underlying provider.
-// It should validate that the provider can be configured, and provide actionable errors in the case
-// it cannot be. Configuration variables can be read from `vars` using the `stringValue` function -
-// for example `stringValue(vars, "accessKey")`.
-func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) error {
```
Comment (guineveresaenger) [inline provider/resources.go:70]: Why was this here? did returning `nil` serve a purpose?

**Review by guineveresaenger — APPROVED** [review]

File: provider/resources.go (hunk 2 above)
Comment (iwahbe) [reply provider/resources.go:70]: It was here from the original provider template, and never served a purpose.

File: provider/resources.go (hunk 1 above)
Comment (iwahbe) [reply provider/resources.go:69]: Sure
